docs/user/troubleshootingbootstrap.md

```diff
@@ -0,0 +1,219 @@
+# Troubleshooting Bootstrap Failures
+
+Unfortunately, there will always be some cases where OpenShift fails to install properly. In these events, it is helpful to understand the likely failure modes as well as how to troubleshoot the failure.
+
+## Gathering bootstrap failure logs
+
+### Using the installer provisioned workflow
+
+When users are using the installer to create the OpenShift cluster, the installer has all the information to automatically capture the logs from bootstrap host in case of failure.
+
+#### Authenticating with bootstrap host for ipi
+
+The installer will use the user's environment to discover the credentials to connect to the bootstrap host over SSH. One of the following methods is used by the installer,
+
+1. Use the user's already setup `SSH_AGENT`. If the user has a ssh-agent setup, the installer will use it for SSH authentication.
+
+2. Use the user'd home directory, `~/.ssh` on Linux hosts, to load all the SSH private keys and use those for SSH authentication.
```

Comment:
`user's`
Also, clarify that this only happens if the SSH_ANGENT isn't already running.

Reply:
> Also, clarify that this only happens if the SSH_ANGENT isn't already running.
https://github.com/openshift/installer/pull/3506/files#diff-135e3d860b56722d4c6282c25380d24dR13 already says `One of`